serverless/cmd/feeder/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:buster AS builder
+
+ARG GOFLAGS=""
+ENV GOFLAGS=$GOFLAGS
+ENV GO111MODULE=on
+
+# Move to working directory /build
+WORKDIR /build
+
+# Copy and download dependency using go mod
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+# Copy the code into the container
+COPY . .
+
+# Build the application
+RUN go build -o /build/bin/feeder ./serverless/cmd/feeder
+COPY ./serverless/cmd/feeder/feed-to-github /build/bin
+
+# Build release image
+FROM golang:buster
+
+RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg \
```

Comment:
I think this could do with some docstring - there's a lot of stuff being piped through a sausage factory here.

Reply:
👍